custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableau2.json

```diff
@@ -1941,6 +1933,77 @@
           "type": "constant",
           "constant": "no"
         }
+      },
+      "tetanus_complete_date": {
+        "comment": "TT Complete date is currently stored on the task case.  Ideally should be stored on the ccs_record case",
+        "type": "conditional",
+        "test": {
+          "type": "not",
+          "filter": {
+            "operator": "in",
+            "type": "boolean_expression",
+            "expression": {
+              "type": "root_doc",
+              "expression": {
+                "datatype": "date",
+                "type": "property_name",
+                "property_name": "tt_complete_date"
+              }
+            },
+            "property_value": [
+              "",
+              null
+            ]
+          }
+        },
+        "expression_if_true": {
+          "type": "root_doc",
+          "expression": {
+            "datatype": "date",
+            "type": "property_name",
+            "property_name": "tt_complete_date"
+          }
+        },
+        "expression_if_false": {
+          "type": "nested",
+          "argument_expression": {
+            "type": "reduce_items",
+            "aggregation_fn": "first_item",
+            "items_expression": {
+              "type": "filter_items",
+              "items_expression": {
+                "type": "get_subcases",
+                "case_id_expression": {
+                  "type": "root_doc",
+                  "expression": {
+                    "type": "property_name",
+                    "property_name": "_id"
+                  }
+                }
+              },
+              "filter_expression": {
```

Comment:
I'm not super familiar with the case structure here. Are there multiple task cases for each ccs record case? Should this filter be based on type instead of a property name? or maybe it can be filtered both on type and property name